test/Bounty.test.js

```diff
@@ -1,10 +1,11 @@
-import { ethGetBalance, ethSendTransaction } from './helpers/web3';
+const { ethGetBalance, ethSendTransaction } = require('./helpers/web3');
+
+var SecureTargetBounty = artifacts.require('SecureTargetBounty');
```

Comment:
Can we use `const` for these?

Reply:
Yes, but we're only removing Babel in this PR, that's coming on a separate one to keep everything tidy.

Reply:
Created #1091.